docs/reference/client-worker-configuration.md

```diff
@@ -14,36 +14,40 @@ The file may be called by any name but should have the following structure:
       "radius": 2
     }
   },
-  "streamingQuery": {
+  "streamingQuery": [
     ...
-  },
+  ],
   "componentDelivery": {
     "default": "RELIABLE_ORDERED",
     "checkoutAllInitially": true
   },
   "componentSettings": {
     ...
   },
-  "permissions": {
+  "permissions": [
```

Comment:
I don't think the permissions field is supposed to be an array

Reply:
This is the format specified in our current docs: https://docs.improbable.io/reference/13.5/shared/worker-configuration/permissions#worker-permissions

Reply:
Fixed. :)